src/post.ts

```diff
@@ -0,0 +1,10 @@
+import {EventPayload} from './EventPayload';
+
+const post = (
+    url: string,
+    body: EventPayload
+) => {
+    return navigator.sendBeacon(url, JSON.stringify(body));
```

Comment:
Are you not able to just pass down `body`? sendBeacon does accept object as it's data param.

Reply:
There's an error if I just pass down `body` :

`Argument of type 'EventPayload' is not assignable to parameter of type 'BodyInit | null | undefined'.`

Not sure if it's just an issue with the IDE, but most examples I'm seeing online stringifies the JSON data (e.g. https://xgwang.me/posts/you-may-not-know-beacon/ or https://www.w3.org/TR/beacon/#example-1), or creates a FormData or Blob object (https://medium.com/@laura.nickerson/javascripts-sendbeacon-de1cc1c97b69)